test/blackbox-tests/test-cases/install-libdir/run.t

```diff
@@ -0,0 +1,15 @@
+`dune install` should handle destination directories that don't exist
+
+  $ dune build @install
+  $ dune install --prefix /tmp --libdir lib
```

Comment:
The test should delete `/tmp/lib` after this. I suggest to use a local directory instead: `--prefix install`.

Reply:
Ah, that's annoying, we'll need to look into this

Reply:
Could you rebase and try again with a relative prefix? I fixed the problem with `Path.mkdir_p`. The output should be stable as it should print a relative path. Writing in `/tmp` using a non random name doesn't feel right.

Reply:
Agree about fixed files. Plus some systems might not have `/tmp`, etc.

Reply:
Thanks for the fix @diml. Will update the test.